crates/biome_json_parser/tests/json_test_suite/allow_trailing_commas/basic.json

```diff
@@ -0,0 +1,7 @@
+{
+    "prop1": [
+        1,
+        2,
+    ],
+    "prop2": 0,
```

Comment:
We should add more cases, so we cover all the possible valid/not valid combinations:

```
true,
```

Valid or not? We should have a case for that
___


```
null,
```

Valid or not? We should have a case for that

___


```
{},
```

Valid or not? We should have a case for that
___